### ISSUE TITLE
Host multiple web assembly in a single asp.net core server host

### ISSUE THREAD

**Issue by akanshanegi** [dotnet/aspnetcore#24272]
I need to host 2 web assembly project in a single asp.net core server.
I am referring to https://github.com/javiercn/BlazorMultipleApps/tree/master/BlazorMultipleApps

But it does not work with WebAssembly 3.2.1
I have also tried adding 2 WASM projects to a single server and the error I am getting is: Blazor static assets - conflicting assets with the same path. I was able to resolve this build error with adding < StaticWebAssetBasePath >.
After adding this I am not getting error but the webassembly razor pages are not getting rendered. 
Any help would be highly appreciated.

**Comment by Jayson-Furr** [comment]
This worked fine for me just a couple of days ago with 3.2.1 Blazor WASM. May want to give it a go with a fresh project. Make sure your mapWhen is setup to work in your development scenario. I used a appSetting to specify which client I was developing on. Then in the mapWhen method I added || (Configuration.Get(name-of-key) == “internalClient) to the one mapped to internal.blah.net. I don’t have the exact code with me but this may help you.
